src/views/Group/CreateGroup.vue

```diff
@@ -0,0 +1,285 @@
+            :key="profile.uuid"
+            v-on:click="addMember(index)"
+          >
+            <div class="pro-box row m-0 align-items-center">
+              <img
+                class="pro-pic rounded-circle"
+                src="@/assets/profile-picture-1.jpg"
+                alt=""
+              />
+              <div class="pro-name pl-3">
+                <p>{{ profile.user.username }}</p>
+              </div>
+            </div>
+            <!-- <img src="@/assets/dtick.png" alt="tick" class="pro-tick" /> -->
+          </div>
+
+          <!-- create button -->
+          <div class="w-100 row m-0 justify-content-center">
+            <button
+              class="
+                col col-5
+                text-center
+                create-btn
+                p-2
+                rounded
+                border-none
+                btn
+              "
+              v-on:click="groupFormed = 1"
+            >
+              Next
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { stringFormat } from "@/helpers";
+import { config } from "@/configurations";
+import { groupService } from "@/services";
+import router from "@/router";
+
+export default {
+  name: "CreateGroup",
+  data() {
+    return {
+      groupName: "",
+      groupDescription: "",
+      members: [],
+      is_paid: false,
+      openAddMembers: 0,
+      profile_info: [],
+      searchValue: "",
+      groupFormed: 0,
+    };
+  },
+
+  watch: {
+    searchValue: function () {
+      this.searchProfile();
+    },
+  },
+
+  methods: {
+    addMember(index) {
+      this.members.push(this.profile_info[index]);
+    },
+
+    removeMember(index) {
+      this.members.splice(index, 1);
+    },
+
+    searchProfile() {
+      const username = this.searchValue;
+      const { dispatch } = this.$store;
+      
+      groupService
+        .searchProfile(username)
+        .then((profile_info) => {
+          this.profile_info = profile_info;
+        })
+        .catch(() => {
+          dispatch(
+            "alertStore/error",
+            config.messagingConfig.messages.error.unknown_error
+          );
+        });
+    },
+
+    createGroup() {
+      const { groupName, groupDescription, members, isPaid } = this;
+      const { dispatch } = this.$store;
+
+      if (!groupName) {
+        dispatch(
+          "alertStore/error",
+          stringFormat(
+            config.messagingConfig.messages.error.empty_field_error,
+            "Groupname",
+            []
+          ).trim(),
+          { root: true }
+        );
+        return;
+      }
+
+      groupService
+        .addGroup({
+          name: groupName,
+          description: groupDescription,
+          is_paid: isPaid,
+          members: members,
+        })
+        .then(() => {
+         router.push({
+           name:"Group"
+         });
+        })
+        .catch(() => {
+          dispatch(
```

Comment:
Use error from service. Applicable to all such occurrences in this PR.

Reply:
Done Sir